appveyor.yml

```
@@ -16,20 +12,23 @@ configuration:
   - Debug
   - Release
 
-environment:
-  matrix:
-  - arch: Win32
-  - arch: Win64
+platform:
+  - x86
+  - x64
 
 init:
-  - echo %APPVEYOR_BUILD_WORKER_IMAGE% - %configuration% - %arch%
+  - echo %APPVEYOR_BUILD_WORKER_IMAGE% - %configuration% - %PLATFORM%
   - if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" (set vs=Visual Studio 15 2017)
   - if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" (set vs=Visual Studio 14 2015)
   - if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2013" (set vs=Visual Studio 12 2013)
-  - if "%arch%"=="Win64" (set generator="%vs% Win64")
-  - if "%arch%"=="Win32" (set generator="%vs%")
+  - if "%PLATFORM%"=="x64" (set generator="%vs% %PLATFORM%")
+  - if "%PLATFORM%"=="x86" (set generator="%vs% Win32")

```

```suggestion
  - if "%PLATFORM%"=="x86" (set generator="%vs%")
```
Excuse me, my mistake.